app/src/main/java/com/codesoom/assignment/application/ProductService.java

```diff
@@ -4,51 +4,85 @@
 import com.codesoom.assignment.domain.Product;
 import com.codesoom.assignment.domain.ProductRepository;
 import com.codesoom.assignment.dto.ProductData;
+import com.github.dozermapper.core.DozerBeanMapperBuilder;
+import com.github.dozermapper.core.Mapper;
 import org.springframework.stereotype.Service;
 
 import javax.transaction.Transactional;
 import java.util.List;
 
+/**
+ * 장난감의 CRUD 를 담당합니다.
+ */
 @Service
 @Transactional
 public class ProductService {
+    private final Mapper mapper;
     private final ProductRepository productRepository;
 
-    public ProductService(ProductRepository productRepository) {
+    /**
+     * 레포지토리 레이어를 주입하여 서비스 객체를 생성합니다.
+     *
+     * @param dozerMapper
+     * @param productRepository 장난감의 레포지토리 객체
+     */
+    public ProductService(Mapper dozerMapper, ProductRepository productRepository) {
+        this.mapper = dozerMapper;
         this.productRepository = productRepository;
     }
 
+    /**
+     * 모든 장난감을 리턴합니다.
+     *
+     * @return 모든 장난감 목록
+     */
```

Comment:
`모든`을 이미 표현하고 있으니 다음과 같이 쓰는 것은 어떤가요?
```suggestion
    /**
     * 모든 장난감들을 리턴합니다.
     *
     * @return 장난감 목록
     */
```